### ISSUE TITLE
explore atom's pathwatcher module

### ISSUE THREAD

**Issue by zaggino** [zaggino/brackets-electron#26]
module is here: https://github.com/atom/node-pathwatcher

atom uses it to watch changes on files, maybe we could use it too; i've no idea how much it differs from  what we have, but maybe we could help improve pathwatcher with features from our `fsevents` and `fsevents-win`

**Comment by zaggino** [comment]
We'll merge master once https://github.com/adobe/brackets/pull/12647 is merged into it to resolve this issue.
